src/main/java/com/iexec/sms/secret/compute/TeeTaskComputeSecret.java

```diff
@@ -77,6 +78,19 @@ public class TeeTaskComputeSecret {
     @Size(min = SECRET_KEY_MIN_LENGTH, max = SECRET_KEY_MAX_LENGTH)
     private String key;
     @NotNull
+    /*
+     * Expected behaviour of AES encryption it to not expand the data very much.
```

Comment:
Typo in `encryption it to not` ?

Reply:
Total typo! Fixed here https://github.com/iExecBlockchainComputing/iexec-sms/pull/105/commits/eaab3995cb8b100f720411528fcaa86b41223302, thanks :)